packages/rpc-core/src/response-patcher-allowed-numeric-values.ts

```diff
@@ -66,6 +66,67 @@ export const ALLOWED_NUMERIC_KEYPATHS: Partial<
     getInflationGovernor: [['initial'], ['foundation'], ['foundationTerm'], ['taper'], ['terminal']],
     getInflationRate: [['foundation'], ['total'], ['validator']],
     getInflationReward: [[KEYPATH_WILDCARD, 'commission']],
+    getMultipleAccounts: [
```

Comment:
😬

One of these two things is true:

1. gzip will crush this and the next developer who comes along will notice that there are two places to update/add account parsers
2. we should consider making this a memoized `getAllowedNumericKeypaths()` method so we can run some logic like:
    ```ts
    const accountParserConfigs = [...];
    return {
        getAccountInfo: accountParserConfigs.map(c => (['value', ...c])),
        getMultipleAccounts: accountParserConfigs.map(c => (['value', KEYPATH_WILDCARD, ...c])),
    }
    ```

Reply:
I think that 2nd option is probably worth doing, it becomes a lot to maintain when there are so many methods returning this parsed data